packages/app/src/cli/models/app/loader.ts

```diff
@@ -346,6 +347,11 @@ async function getProjectType(webs: Web[]): Promise<'node' | 'php' | 'ruby' | un
 }
 
 async function logMetadataForLoadedApp(app: App) {
-  const projectType = await getProjectType(app.webs)
-  metadata.addPublic({project_type: projectType})
+  try {
+    const projectType = await getProjectType(app.webs)
+    metadata.addPublic({project_type: projectType})
+    // eslint-disable-next-line no-catch-all/no-catch-all
+  } catch (error) {
+    output.debug(`Unable to logMetadataForLoadedApp ${error}`)
```

Comment:
This would swallow legit bugs that we want to see reported to Bugsnag.

Reply:
It would, though not bugs we want to justify crashing for. Mentioned to Isaac, but will add something for doing soft-reporting to Bugsnag for these scenarios, where we can still proceed with execution. But not in this branch - got to get the fix out soon.

Reply:
It's not a big deal if we go ahead with merging this as it is, but I've seen projects taking this stance too far and turning many  the bugs into "not worth the crash". I'd lean on the side of letting errors happen and having the right systems in place (i.e. tests) to catch those early.